src/lib/style/vendors/_normalize.scss

```diff
@@ -0,0 +1,386 @@
+/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Is this all new, or was it moved?

Reply:
It is new.  I was a little hesitant about it. There was a bunch of stuff in it that I have removed. On once hand it's generally a good idea to have a reset/normalize so that there is a common baseline, but it's a little risky with preexisting codebase. I left things that looked 'safe' to me. But I am not adverse to removing it entirely